parachain/runtime/kintsugi/src/lib.rs

```diff
@@ -1073,6 +1079,7 @@ construct_runtime! {
         Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>} = 5,
         Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 6,
         Identity: pallet_identity::{Pallet, Call, Storage, Event<T>} = 7,
+        Sudo: pallet_sudo::{Pallet, Call, Storage, Config<T>, Event<T>} = 8,
```

Comment:
Worth checking if adding the sudo pallet before the currencies pallet might mess with the ledger app.

Reply:
It should be fine, the Ledger app uses the correct index of the `Tokens` pallet: https://github.com/Zondax/ledger-interlay/blob/685f64613ec736d7290cf706745d50e50e6bc311/app/src/substrate/substrate_dispatch_V2.c#L116

Reply:
The pallet indices are not changing so this change should not be breaking anything